packages/containers/src/SubHeaderBar/SubHeaderBar.container.js

```diff
@@ -110,13 +110,14 @@ class SubHeaderBar extends React.Component {
 
 	render() {
 		const state = this.props.state || DEFAULT_STATE;
+		const hasGoBack = this.props.onGoBack || this.props.actionCreatorGoBack;
 		const props = Object.assign({}, omit(this.props, cmfConnect.INJECTED_PROPS), {
 			editMode: state.get('editMode', false),
 			onEdit: this.onEdit,
 			onCancel: this.onCancel,
 			onSubmit: this.onSubmit,
 			onChange: this.onChange,
-			onGoBack: this.onGoBack,
+			onGoBack: hasGoBack && this.onGoBack,
```

Comment:
That's a lot of ways to set `goBack` 😞

Reply:
hasGoBack is to be able to desactivate the button. Not sure I understand your point.